ide/static/css/zoo.css

```diff
@@ -2,7 +2,7 @@ body {
     overflow-x: hidden;
  }
 
-#wrapper {
+#content-wrapper {
```

Comment:
@rahulsnkr didn't get this, why would changing the id of the css class worked for you?

Reply:
https://github.com/Cloud-CV/Fabrik/blob/0bb88a50f570bf443339d4d4f88c66241412b905/ide/static/js/pane.js#L457-L466

The 'Wrapper' section in the pane.js file also has the same id, so the padding was being applied to that too. So I changed the id of the div in modelZoo.js, and the corresponding property name in zoo.css from 'wrapper' to 'content-wrapper'